src/Aspire.Hosting/ApplicationModel/ContainerImageAnnotation.cs

```diff
@@ -58,4 +58,13 @@ public string? SHA256
             _tag = null;
         }
     }
+
+    /// <summary>
+    /// Gets or sets the target platform for the container image (e.g., "linux/amd64", "linux/arm64").
+    /// </summary>
+    /// <remarks>
+    /// When set, the container runtime will pull and run the image for the specified platform.
+    /// This is useful for multi-architecture images where you want to explicitly select a specific platform variant.
+    /// </remarks>
+    public string? Platform { get; set; }
```

Comment:
It feels like this should be unified with the `ContainerTargetPlatform` from `Aspire.Hosting.Publishing` to avoid having separate run and publish platform config.